AdobeStockAsset/Model/System/Config/Comment.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\AdobeStockAsset\Model\System\Config;
+
+use Magento\Config\Model\Config\CommentInterface;
+use Magento\Framework\UrlInterface;
+
+/**
+ * Class Comment
+ * @package Magento\AdobeStockAsset\Model\System\Config
+ */
+class Comment implements CommentInterface
+{
+    /**
+     * @var UrlInterface
+     */
+    private $url;
+
+    /**
+     * Comment constructor.
+     * @param UrlInterface $url
+     */
+    public function __construct(
+        UrlInterface $url
+    ) {
+        $this->url = $url;
+    }
+
+    /**
+     * @param string $elementValue
+     * @return string
+     */
+    public function getCommentText($elementValue)
+    {
+        $redirect_url = $this->url->getUrl('adobe_stock/oauth/callback');
+        $redirect_url_pattern = $this->url->getUrl('admin/adobe_stock/oauth/callback/');
```

Comment:
Even though I have added `admin` in the redirect_url_pattern but it showing
![image](https://user-images.githubusercontent.com/39480008/62391059-729fe180-b581-11e9-8089-191e7d24b24c.png)

Reply:
Please use camelcase for variables names, i.e. `$redirectUrl`.

Redirect url pattern requires escaping, I think you can simply take `$redirectUrl` and add `\\` before each dot, i.e. `$redirectUrlPattern = str_replace('.', '\\.', $redirectUrl)`.

Reply:
sorry, there is no need to add `admin` to the redirect pattern, that was my typo